src/core/render/emoji-data.js

```diff
@@ -1,4 +1,4 @@
- 
+/* eslint-disable */
```

Comment:
Looks like this file was checked in by accident. No need to include the updated emoji short codes with this PR.

Reply:
I merged it here on purpose, I wanted to modify the emoji action.

This file does not require `/* eslint-disable */`?

Reply:
I believe this is correct now that `eslint-data.js` is ignored via `/eslint.config.js`. The comment is being added by the `build/emoji.js` script while auto-generating `emoji-data.js` content.